papergrid/src/lib.rs

```diff
@@ -797,6 +811,7 @@ impl AlignmentVertical {
 pub struct Settings {
     text: Option<String>,
     padding: Option<Padding>,
+    margin: Option<Margin>,
```

Comment:
I think `Margin` doesn't belong to `Settings` because settings are used to configure a segment of table, whether margin affects a table as a whole.

So it's better to create a `set_margin` method on `Grid` level I think.

Reply:
As I see it's actually unused currently?

Reply:
And seems like you've already created `set_margin`

Reply:
sorry, 
At first I added `Margin`  to `Settings`. After some refactoring, I forgot to remove `Margin` from `Settings` ....